MoEPluginInbox/MoEInboxPluginController.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  MoEInboxPluginController.swift
+//  MoEPluginInbox
+//
+//  Created by Rakshitha on 27/06/22.
+//
+
+import Foundation
+import MoEngageInbox
+
+class MoEInboxPluginController {
+    var identifier: String
```

Comment:
@RakshithaAcharya Same here, please make sure you use right access specifier

Reply:
@deepa-iOS-SDK done